runcheckmigration

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+checkmigrations --migrations-to-ignore 3 \
```

Comment:
we don't need to ignore migrations here, remove `--migrations-to-ignore 3 \` and put the command on one line only